subsys/bluetooth/controller/Kconfig.ll_sw_split

```
@@ -189,6 +189,22 @@ config BT_CTLR_ADV_SYNC_SET
 	help
 	  Maximum supported periodic advertising sets.
 
+config BT_CTLR_ADV_DATA_CHAIN
+	bool "Advertising Data chaining"
```

We've typically added "[EXPERIMENTAL]" for experimental Kconfigs

```suggestion
	bool "Advertising Data chaining [EXPERIMENTAL]"
```